lib/Sabberworm/CSS/Parsing/OutputException.php

```diff
@@ -7,12 +7,15 @@
 */
 class OutputException extends \Exception {
     private $iLineNo;
-    public function __construct($sMessage, $iLineNo = 0)
-    {
+    public function __construct($sMessage, $iLineNo = 0) {
         $this->$iLineNo = $iLineNo;
         if (!empty($iLineNo)) {
             $sMessage .= " [line no: $iLineNo]";
         }
         parent::__construct($sMessage);
     }
+
+    public function getLineNo() {
+        return $this->getLine();
```

Comment:
I meant for the CSS line numner (`$iLineNo` member), not for the line number of the PHP code that threw the exception.